task_DEV_6/task_DEV_6/CustomStack.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+
+namespace task_DEV_6
+{
+    class CustomStack
+    {
+        private int[] arrayStack;
+        private int count;
+        private const int ARRAYLENGTH = 10;
+
+        public CustomStack()
+        {
+            arrayStack = new int[ARRAYLENGTH];
+        }
+
+        public CustomStack(int item)
```

Comment:
`item`? More like `capacity` or smth like this.

Reply:
Done.